include/deal.II/dofs/dof_tools.h

```diff
@@ -1107,7 +1128,16 @@ namespace DoFTools
     const ComponentMask &            component_mask = ComponentMask(),
     const std::vector<unsigned int> &first_vector_components =
       std::vector<unsigned int>(),
-    const number periodicity_factor = 1.);
+    const number periodicity_factor = 1.)
+  {
+    make_periodicity_constraints<DoFHandlerType::dimension,
+                                 DoFHandlerType::space_dimension>(
+      periodic_faces,
+      constraints,
+      component_mask,
+      first_vector_components,
+      periodicity_factor);
+  }
```

Comment:
Can you move the implementation to the end of this file like we do in the other functions of this file?